app/imports/components/apply/form-test.jsx

```diff
@@ -9,9 +9,25 @@ import { FormSchema } from '/imports/schemas/TestForm.js';
 
 // Import methods
 import { saveApplicationSection } from '/imports/api/methods.js';
+import { saveApplyPosition } from '/imports/api/methods.js';
 
 // Component
 export default class FormTest extends Component {
+  savePosition() {
+    saveApplyPosition.call({
+      position: this.props.type,
```

Comment:
This might be confusing naming. @varffs plz help. `type` is the application section being loaded. Maybe a better could be `step` ?

In the collecction is is stored as `position`